tools/detection2d/deploy_yolox_s_opt.py

```diff
@@ -0,0 +1,252 @@
+import argparse
```

Comment:
Please do not place standalone algorithm scripts such as `tools/detection2d/deploy_yolox_s_opt.py` or items like those in `tools/detection2d` under the tools directory.
If the tools only use for `YOLOX`, would you move into `projects/YOLOX_elan`?

Also, please refer to the following document for more details:
https://github.com/tier4/AWML/blob/main/docs/design/autoware_ml_design.md